### PR TITLE
Test code coverage

### PR DESCRIPTION
**Pull request by jvdanilo**
Here is the 100% test code coverage.

### CODE REVIEW

**Comment by Anahkiasen** [comment]
Wow, thanks :o Why static test cases though ?

**Comment by jvdanilo** [comment]
I had some error with setUp method where you could run only one method and the next one even correct one would give an errror.

I know that I should no use static methods but I will try and correct that.

**Comment by jvdanilo** [comment]
Here it is without static.

File: tests/PolyglotDatabaseTest.php [hunk 1]
```diff
@@ -2,40 +2,108 @@
...
-use PHPUnit_Framework_TestCase;
-use Illuminate\Events\Dispatcher;
-use Illuminate\Container\Container;
-use Illuminate\Support\Facades\Lang;
-use Polyglot\PolyglotServiceProvider;
-use Illuminate\Support\Facades\Config;
-use Illuminate\Database\Capsule\Manager as Capsule;
+use Illuminate\Database\Eloquent\Model;
 
 class RealArticle extends Polyglot {
```
Comment (Anahkiasen) [inline tests/PolyglotDatabaseTest.php]: Could you move that class to the Dummies namespace ?

File: tests/PolyglotDatabaseTest.php [hunk 2]
```diff
@@ -2,40 +2,108 @@
...
+		$a = new RealArticle();
+		$a->name = 'Start name';
+		$a->title = 'Start title';
+		$a->lang = 'fr';
+		$a->save();
+
+		$article = RealArticle::first();
+		$start = $article->updated_at;
+
+		sleep(1);
```
Comment (Anahkiasen) [inline tests/PolyglotDatabaseTest.php:20]: What was the reason for the `sleep` here ?

File: tests/TestCases/DatabaseTestCase.php [hunk 3]
```diff
@@ -14,9 +15,12 @@
 class DatabaseTestCase extends PHPUnit_Framework_TestCase {
```
Comment (Anahkiasen) [inline tests/TestCases/DatabaseTestCase.php]: Could you tweak the class's style to make it match PSR2-tabs ? Braces for classes and methods on a new line, visibility prefixing methods (`public function __construct`) and stuff ?

Comment (jvdanilo) [inline tests/TestCases/DatabaseTestCase.php]: Yes of course

File: tests/PolyglotDatabaseTest.php (hunk 2 above)
Comment (jvdanilo) [inline tests/PolyglotDatabaseTest.php:20]: Field updated_at is updated on save. Without sleep the time would be the same because test runs too fast.

Comment (Anahkiasen) [inline tests/PolyglotDatabaseTest.php:20]: Kay, got it.

**Comment by jvdanilo** [comment]
Here are the changes you requested, I hope i did everything right.

**Comment by Anahkiasen** [comment]
Oh, uh... actually I was talking about PSR2-tabs, so, the opposite :p I'm so sorry, if you don't feel like reverting I'll just merge this and try to get some time later to repass the files to tabs.

**Comment by jvdanilo** [comment]
Sory, I have commited the reverse now.

**Comment by Anahkiasen** [comment]
Allright looks perfect, thanks a _lot_ for your contribution!

**Pull request merged**